__init__.py

```diff
@@ -1,10 +0,0 @@
-# initialize this directory as a module, provide its in your Python path
-
-import warnings
```

Comment:
With this, all of the warnings we probably ought to have been seeing are made visible.